pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
In `pkg/getty/docs/` it won't work, because it is under `pkg/getty/` (or `pkg/<whatever>/`), so you mean the main `docs/`? Fine, but I do think each `pkg/<whatever>/` should have its own `README.md`. 

So is the structure:

```
|- pkg
      |- whatever 
             |- README.md
             |- image/
                  |- Makefile
                  |- Dockerfile
                  |- etc/
 ....
```

or just to keep the `README.md` in scope, but not reference an actual hash?